docs/root/operations/runtime.rst

```diff
@@ -6,3 +6,9 @@ Runtime
 :ref:`Runtime configuration <config_runtime>` can be used to modify various server settings
 without restarting Envoy. The runtime settings that are available depend on how the server is
 configured. They are documented in the relevant sections of the :ref:`configuration guide <config>`.
+
+Runtime flags are also used as a mechanism to disable new behavior or risky changes: such changes
+will tend to introduce a runtime flag that can be used to disable the new behavior/code path. As such
```

Comment:
I'm 99% likely to blame for calling them flags, but do we want to call them flags?  I've been calling them runtime guards in GOVERNANCE.md

Reply:
+1 I would use runtime guards/features to match the rest of the docs.